recipes-domd/agl/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7796-salvator-x-domd.dts

```diff
@@ -986,14 +991,14 @@
 	status = "okay";
 };
```

Comment:
Why IPMMUs are still enabled in domd device tree? They must be disabled or even dropped. Moreover, Xen doesn't insert IOMMU in dom0 device tree even it is not owned by Xen.

Reply:
removed